### DISCUSSION TITLE
Limit number of PR's

### DISCUSSION THREAD

**Discussion by rowbot-99** [rfennell/AzurePipelines#1354] [Q&A]
Hi,

I have noticed that when I create a new branch off my main branch, it seems to list around 29 PR's when I run the pipeline against the new branch. 

How do I show a PR only if it's run against the main branch (where the PR's happen) and not branches that are from the main branch since we didn't merge anything into those new branches.

Thanks

**Comment by rfennell** [comment]
Unless within the PR loop in the template you could put a check against the branch the commit was to (not sure if this is available) I don;t see a solution to your question
